index.html

```diff
@@ -5,51 +5,224 @@
+                        </div>
+                    </section>
+                    <div class="contact">
+                        <h3 class="am-connect">LET'S CONNECT</h3>
+                    </div>
+                    <div class="connect-logo">
+                        <div class="logo">
+                            <ul class="logo-div">
+                                <li><a href=""><img src="./image/twiter.svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector.png" alt=""></a></li>
+                                <li><a href=""><img src="./image/M.svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector(3).svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector(4).svg" alt=""></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="see-project">
+                        <button type="button">Get my resume</button>
+                    </div>
+                        <div class="am-menu">
+                            <div class="languages">
+                                <div class="lang-menu">
+                                    <div class="lan-text">Languages</div>
+                                    <div class="lang-logo"><img src="./image/down_arrow.png" alt=""></div>
+                                </div>
+                                <div class="javascript">
+                                    <div class="ln-image"><img src="./image/javascript.png" alt=""></div>
+                                    <div class="ln-text">JavaScript</div>
+                                </div>
+                                <div class="html">
+                                    <div class="ln-image"><img src="./image/html.png" alt=""></div>
+                                    <div class="ln-text">HTML</div>
+                                </div>
+                                <div class="css">
+                                    <div class="ln-image"><img src="./image/css.png" alt=""></div>
+                                    <div class="ln-text">CSS</div>
+                                </div>
+                            </div>
+                            
+                            <div class="frame">
+                                <nav>
+                                    <div>Frameworks</div>
+                                    <div><img src="./image/right_arrow.png" alt=""></div>
+                                </nav>
+                            </div>
+                            <div class="skills">
+                                <nav>
+                                    <div>Skills</div>
```

Comment:
- [ ] As suggested by the previous reviewer, please kindly remove the `<nav>` tag surrounding your skills set.

Reply:
Removed the same.